src/excmds.ts

```diff
@@ -3129,9 +3126,8 @@ function parse_bind_args(...args: string[]): bind_args {
         - [[reset]]
 */
 //#background
-export function bind(...args: string[]) {
```

Comment:
These are just some minor changes to typecheck with new return types of some functions. This has to be async to have a uniform interface with the shim generated for content scripts

Reply:
Can they be a separate PR? Doesn't look like it has much to do with the messaging.